Python/bytecodes.c

```diff
@@ -4840,6 +4840,10 @@ dummy_func(
             assert(((_PyExecutorObject *)executor)->vm_data.valid);
         }
 
+        tier2 op(_SET_EXECUTOR_RUN_STATE, (--)) {
```

Comment:
Not a big deal, but I find the name a tiny bit wordy and not super descriptive. Maybe `_MAKE_WARM` or `_WARM_UP` or something?